ephios/api/views/users.py

```diff
@@ -103,8 +104,8 @@ class UserByMailView(RetrieveModelMixin, GenericViewSet):
 class UserParticipationView(viewsets.ReadOnlyModelViewSet):
     serializer_class = ParticipationSerializer
     permission_classes = [IsAuthenticatedOrTokenHasScope]
-    filter_backends = [ObjectPermissionsFilter, DjangoFilterBackend]
-    filterset_fields = ["state"]
+    filter_backends = [ParticipationPermissionFilter, DjangoFilterBackend]
+    filterset_fields = ["state", "shift__event__type"]
```

Comment:
@felixrindt this means that users have to use `?shift__event__type=3` as the query param. Are we fine with that or should we try to work around it?

Reply:
I think we should get used to building custom Filtersets. I think it will be important for Viewsets for user, shifts, events, participations etc as well:

https://django-filter.readthedocs.io/en/latest/guide/rest_framework.html